src/Mapper.cpp

```diff
@@ -1346,3 +1346,206 @@ map<int, CGRANode*>* Mapper::getReorderPath(map<CGRANode*, int>* t_path) {
   return reorderPath;
 }
 
+// Saves the mapping results to json file for subsequent incremental mapping
```

Comment:
Period at the end, thanks! Same to the others.

Reply:
Sorry what do you mean?

Reply:
`mapping` -> `mapping.` Sorry for being picky.

Reply:
Has fixed all similar issues